src/app/js/root-admin/Tenants.js

```diff
@@ -267,15 +270,43 @@ const Tenants = ({ handleLogout }) => {
                                 background: '#D9F3FF',
                                 textDecoration: 'none',
                             },
-                            '&:active': {
-                                background: '#0069A9',
-                                color: '#D9F3FF',
-                            },
                         }}
-                        title={params.value}
                     >
-                        {params.value}
-                    </Link>
+                        <Link
+                            href={`${baseUrl}/instance/${name}`}
+                            target="_blank"
+                            title={params.value}
+                            color="primary"
+                            sx={{
+                                overflow: 'hidden',
+                                textOverflow: 'ellipsis',
+                                whiteSpace: 'nowrap',
+                                outline: 'none',
+                                textDecoration: 'none',
+                                height: '100%',
+                                width: '100%',
+                                display: 'flex',
+                                alignItems: 'center',
+                                padding: '0 16px',
+                                '&:focus': {
+                                    textDecoration: 'none',
+                                },
+                                '&:hover': {
+                                    textDecoration: 'none',
+                                },
+                            }}
+                        >
+                            {params.value}
+                        </Link>
+                        <IconButton
+                            size="large"
+                            href={`${baseUrl}/instance/${name}/admin`}
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
**praise:** ‏Good idea!